docs/architecture/decisions/0003-cross-service-authorisation.md

```diff
@@ -0,0 +1,23 @@
+# 3. Cross-service authorisation
+
+Date: 2024-03-12
+
+## Status
+
+Accepted
+
+## Context
+
+When an LPA is created or updated in the LPA Store, we need to keep a record of who the initiator of the action is. This ensures we have an audit trail of all changes, whether they come from a member of the public, a member of staff at the OPG or an internal service.
+
+## Decision
+
+Requests to create or update an LPA will include a URN identifier to point to the soruce of the action. This will be in the format: `urn:opg:${service}:users:${identifier}`.
+
+For **members of the public using Make and Register**, the `service` will be "poas:mrlpa" and the `identifier` will be the actor's unique ID as stored on the LPA document (see ADR 2).
```

Comment:
we are currently using `poas:makeregister` for this. Also would it be nice to map these to what is used by events, so `poas:sirius` instead?

Reply:
The problem we have is that Sirius-level auth isn't specific to POAS or Supervision. With an eye to make this work more globally than just in modernise processes, using `poas:sirius` would tie us in knots a bit.